sklearn/tests/test_naive_bayes.py

```diff
@@ -284,8 +286,8 @@ def test_discretenb_pickle(cls):
     assert_array_equal(y_pred, clf2.predict(X2))
 
 
-@pytest.mark.parametrize('cls', [BernoulliNB, MultinomialNB, GaussianNB,
-                                 CategoricalNB])
```

Comment:
Maybe renaming the `discrete` part would be good since we have the `GaussianNB`

Reply:
Done in 6e51074